Source/WebKit/Platform/cocoa/WebPrivacyHelpers.h

```diff
@@ -36,6 +36,9 @@
 #if ENABLE(ADVANCED_PRIVACY_PROTECTIONS)
 #import <WebCore/LinkDecorationFilteringData.h>
 #import <WebCore/OrganizationStorageAccessPromptQuirk.h>
+#ifdef __OBJC__
+#import <pal/spi/cocoa/WebPrivacySPI.h>
+#endif
```

Comment:
Wouldn't we unconditionally have `__OBJC__` under `Platform/cocoa/`?

Reply:
Ah, so the header is actually included in `.cpp` (non-ObjC) files behind Cocoa-specific platform or feature flags.